src/utils/futures.rs

```diff
@@ -172,3 +249,79 @@ impl Drop for CallOnDrop {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    use crate::test;
+
+    #[test]
+    fn test_remote_thread() {
+        // Ensure actix is setup.
+        test::setup();
+
+        // test::setup() enables logging, but this test spawns a thread where
+        // logging is not captured.  For normal test runs we don't want to
+        // pollute the stdout so silence logs here.  When debugging this test
```

Comment:
tests suppress stdout by default, so I‘m not sure you need to silence them manually?

Reply:
Because we spawn a new thread and in new threads this is not done, and the functionality to do this isn't even exposed as a public API.  So we end up with this sad thing.